AhmadDuhoki/css/style.css

```diff
@@ -0,0 +1,195 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html, body{
+    background-color: black;
+    color: #C0C0C0;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.logo{
+    width: 70px;
+    height: 65px; 
+    margin-top: 8px;
+    margin-left: 50px;
+}
+
+.header {
+    position: fixed;
+    top: 0;
+    z-index: 1;
+    width: 100%;
+    height: 80px;
+    background-color: black;
+    border-radius: 4px;
+    -webkit-box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+    -moz-box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+    box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+}
+
+.light-btn{
+    border: 2px solid #FFA500;
+    padding: 6px 12px;
+    background-color: black;
+    color: #FFA500;
+    border-radius: 5px;
+}
+
+.light-btn:hover{
+    color: white;
+}
+
+.dark-btn{
+    border: 2px solid #FFA500;
+    padding: 6px 12px;
+    background-color: #FFA500;
+    color: black;
+    border-radius: 5px;
+}
+
+.dark-btn:hover{
+    color: white;
+}
+
+.btns {
+    float: right;
+    margin-top: 30px;
+    margin-right: 50px;
+}
+
+a{
+    text-decoration: none;
+    text-transform: uppercase;
+    color: white;
+}
+
+#wrapper{
+    width: 75%;
+    height: 100vh;
+    margin: 0 auto;
+    background-color: black;
+    -webkit-box-shadow: 0px 0px 0px 1px rgba(255,166,0,1);
+    -moz-box-shadow: 0px 0px 0px 1px rgba(255,166,0,1);
+    box-shadow: 0px 0px 0px 1px rgba(255,166,0,1);
+}
+
+#background-pic{
+    position: relative;
+    width: 100%;
+    height: 550px;
+    background-image: url(/AhmadDuhoki/imgs/demonSlayer.jpg);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.profile-pic{
+    position: absolute;
+    bottom: -100px;
+    width: 200px;
+    height: 200px;
+    border: 3px solid #FFA500;
+    border-radius: 50%;
+    margin-left: 80px;
+}
+
+#name{
+    position: absolute;
+    top: 60%;
+    left: 30%;  
+}
```

Comment:
Your webpage is not looking consistent and a reason is that you are using absolute positioning to position blocks within your webpage. It makes sense to use it for the profile pic but not for everything and I know this was done within the breakout room when testing people's suggestions. I just wanted to let you know.